app/test_main.py

```diff
@@ -1,3 +1,29 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected_output",
+    [
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 28, [2, 2]),
+        (28, 29, [3, 3])
```

Comment:
The test cases seem to be testing similar scenarios multiple times. According to the checklist, 'Don't Repeat Yourself', ensure that your tests don't test the same case multiple times. Consider removing redundant test cases.